gitbook/tables/time-signatures.html

```diff
@@ -2361,75 +2361,213 @@ <h2 id="recommended-stylistic-alternates">Recommended stylistic alternates</h2>
+<td><strong>uniE088.salt01</strong><br><em>timeSig8Large</em><br>Time signature 8 (outside staff)</td>
 <td style="text-align:center"><span class="bravura_large">&#xF465;</span></td>
-<td><strong>uniE088.ss01</strong><br><em>timeSig8Small</em><br>Time signature 8 (small staff)</td>
+<td><strong>uniE088.salt02</strong><br><em>timeSig8Small</em><br>Time signature 8 (small staff)</td>
 </tr>
 <tr>
+<td style="text-align:center"><span class="bravura_large">&#xF50E;</span></td>
+<td><strong>uniE088.salt03</strong><br><em>timeSig8Narrow</em><br>Time signature 8 (large, narrow)</td>
 <td style="text-align:center"><span class="bravura_large">&#xF449;</span></td>
-<td><strong>uniE089.ss04</strong><br><em>timeSig9Large</em><br>Time signature 9 (outside staff)</td>
+<td><strong>uniE089.salt01</strong><br><em>timeSig9Large</em><br>Time signature 9 (outside staff)</td>
+</tr>
+<tr>
 <td style="text-align:center"><span class="bravura_large">&#xF466;</span></td>
-<td><strong>uniE089.ss01</strong><br><em>timeSig9Small</em><br>Time signature 9 (small staff)</td>
+<td><strong>uniE089.salt02</strong><br><em>timeSig9Small</em><br>Time signature 9 (small staff)</td>
+<td style="text-align:center"><span class="bravura_large">&#xF50F;</span></td>
+<td><strong>uniE089.salt03</strong><br><em>timeSig9Narrow</em><br>Time signature 9 (large, narrow)</td>
 </tr>
 <tr>
 <td style="text-align:center"><span class="bravura_large">&#xF44A;</span></td>
-<td><strong>uniE08A.ss04</strong><br><em>timeSigCommonLarge</em><br>Common time (outside staff)</td>
+<td><strong>uniE08A.salt01</strong><br><em>timeSigCommonLarge</em><br>Common time (outside staff)</td>
+<td style="text-align:center"><span class="bravura_large">&#xF510;</span></td>
+<td><strong>uniE08A.salt02</strong><br><em>timeSigCommonNarrow</em><br>Common time (large, narrow)</td>
+</tr>
+<tr>
 <td style="text-align:center"><span class="bravura_large">&#xF44B;</span></td>
-<td><strong>uniE08B.ss04</strong><br><em>timeSigCutCommonLarge</em><br>Cut time (outside staff)</td>
+<td><strong>uniE08B.salt01</strong><br><em>timeSigCutCommonLarge</em><br>Cut time (outside staff)</td>
+<td style="text-align:center"><span class="bravura_large">&#xF511;</span></td>
+<td><strong>uniE08B.salt02</strong><br><em>timeSigCutCommonNarrow</em><br>Cut time (large, narrow)</td>
 </tr>
 <tr>
 <td style="text-align:center"><span class="bravura_large">&#xF44C;</span></td>
-<td><strong>uniE08C.ss04</strong><br><em>timeSigPlusLarge</em><br>Time signature + (outside staff)</td>
-<td style="text-align:center">&#xA0;</td>
-<td>&#xA0;</td>
+<td><strong>uniE08C.salt01</strong><br><em>timeSigPlusLarge</em><br>Time signature + (outside staff)</td>
+<td style="text-align:center"><span class="bravura_large">&#xF512;</span></td>
+<td><strong>uniE08C.salt02</strong><br><em>timeSigPlusNarrow</em><br>Time signature + (large, narrow)</td>
+</tr>
+<tr>
+<td style="text-align:center"><span class="bravura_large">&#xF4EE;</span></td>
```

Comment:
I don't see these new stylistic variants listed in the version history. They appear on my system as clef and other notation symbols.

Reply:
I was asking because once a stylistic alternate is listed with a unique SMuFL name, it becomes something that MusicXML files can use in smufl attributes and elements. We don't need to change anything for this pull request, but it might warrant further discussion.